objective-c.html.markdown

```diff
@@ -70,7 +70,7 @@ int main (int argc, const char * argv[])
     char theLetterZ            = [theLetterZNumber charValue]; // or 'Z'
     NSLog(@"%c", theLetterZ);
 
-    // Integral literals
```

Comment:
For this line, I like the old wording better. "Integer" would be ambiguous here, since it can take the mathematical sense (and indeed all the literals below are integers), or the programming sense of `int`.